cdap-docs/_common/common-build.sh

```diff
@@ -337,10 +247,10 @@ function download_file() {
   local file_name=${3}
   local md5_hash=${4}
   local target_filename=${5}
-  if [[ "x${target_filename}" == "x" ]]; then
-    local target=${includes_dir}/${file_name}
-  else
+  if [[ -n ${target_filename} ]]; then
     local target=${includes_dir}/${target_filename}
+  else
+    local target=${includes_dir}/${file_name}
   fi
```

Comment:
FYI: this whole thing can be done in bash without conditionals...
`local target=${includes_dir}/${target_filename:-${file_name}}`

I'll leave it up to you if you'd like to change these, but generally this sort of comparison makes more sense to do in pure shell, as `[` is often an external symlink to the external `test` command versus the shell built-in...